index.html

```diff
@@ -685,6 +714,22 @@ <h3>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
 </pre>
             </section>
         </section>
+        <section>
+            <h2><code>DeviceProperties</code> interface</h2>
+            <pre class="idl">
+                [Exposed=PointerEvent]
```

Comment:
[Exposed=PointerEvent] is not valid webidl.

Reply:
Thanks! I've changed it to Window. I overlooked the fact that PointerEvent is not a "global" interface.